hydra-configs-torchvision/tests/test_instantiate_transforms.py

```diff
@@ -147,7 +147,7 @@ def identity(x):
             {},
             [],
             {"transforms": [ToTensor()]},
-            transforms.transforms.RandomApply([ToTensor()]),
+            transforms.transforms.RandomApply([ToTensor()], p=0.5),
```

Comment:
The p=0.5 is odd here. add a comment?

Reply:
it was unnecessary for the test. I removed it.